packages/wonder-blocks-dropdown/components/multi-select-menu.js

```diff
@@ -0,0 +1,227 @@
+    static defaultProps = {
+        alignment: "left",
+        disabled: false,
+        displaySelectionShortcuts: false,
+        light: false,
+    };
+
+    constructor(props: Props) {
+        super(props);
+        this.state = {
+            open: false,
+        };
+    }
+
+    toggleMenu() {
+        this.setState((prevState) => ({
+            open: !prevState.open,
+        }));
+    }
+
+    handleSelected(selectedValue: string, selectionState: boolean) {
+        const {onChange} = this.props;
+
+        onChange([{value: selectedValue, selected: selectionState}]);
+    }
+
+    handleSelectAll() {
+        const {items, onChange, selectedItems} = this.props;
+        const update = items
+            .filter((item) => !selectedItems.includes(item.value))
+            .map((item) => {
+                return {
+                    value: item.value,
+                    selected: true,
+                };
+            });
+        onChange(update);
+    }
+
+    handleSelectNone() {
+        const {items, onChange, selectedItems} = this.props;
+        const update = items
+            .filter((item) => selectedItems.includes(item.value))
+            .map((item) => {
+                return {
+                    value: item.value,
+                    selected: false,
+                };
+            });
+        onChange(update);
+    }
+
+    render() {
+        const {
+            alignment,
+            displaySelectionShortcuts,
+            disabled,
+            items,
+            light,
+            placeholder,
+            selectItemType,
+            selectedItems,
+            style,
+        } = this.props;
+
+        const {open} = this.state;
+
+        // TODO(sophie): figure out how to configure plurals for i18n
+        const menuText =
+            selectedItems.length === 0 && placeholder
+                ? placeholder
+                : `${selectedItems.length} ${selectItemType}`;
+
+        const opener = (
+            <SelectBox
+                disabled={disabled}
+                light={light}
+                onClick={() => this.toggleMenu()}
+                style={style}
+            >
+                {menuText}
+            </SelectBox>
+        );
+
+        const menuItems = items.map((item, index) => {
+            return (
+                <SelectItem
+                    disabled={item.disabled}
+                    key={item.value}
+                    label={item.label}
+                    onClick={item.onClick}
+                    onToggle={(value, state) =>
+                        this.handleSelected(value, state)
+                    }
+                    selected={selectedItems.includes(item.value)}
+                    value={item.value}
+                    variant={"checkbox"}
+                />
+            );
+        });
+
+        if (displaySelectionShortcuts) {
+            // TODO(sophie): translate for i18n
```

Comment:
I think as long as we have a version of `i18n._()` that just returns the string passed in we should be good.  @jeresig that should work right?  We should probably add a function to `wonder-blocks-core`.  That can be a separate PR.